CONTRIBUTING.rst

```diff
@@ -286,6 +286,10 @@ To ensure your code meets minimum code styling standards, run::
   pip install pre-commit
   pre-commit run --all-files
 
+If you have issues related to `setuptools` when installing `pre-commit`, see 
+[pre-commit Issue #2178 comment](https://github.com/pre-commit/pre-commit/issues/2178#issuecomment-1002163763)
+for a potential resolution.
```

Comment:
This needs to be ReSTified instead of markdownified, I'm afraid :)

Reply:
My bad. Fixed as requested in https://github.com/pyvista/pyvista/pull/2709/commits/14db08f6b8b31d54efa63d2b68f7de427c34d894.